controller/konnect/ops/ops_kongconsumer.go

```diff
@@ -118,6 +141,192 @@ func updateConsumer(
 	return nil
 }
 
+// handleConsumerGroupAssignments resolves ConsumerGroup references of a KongConsumer, reconciles them with Konnect, and
```

Comment:
Given how many round trips this is going to be and the fact that we have an issue open for declarative consumer group membership API: KOKO-1952, can we add a comment somewhere that this approach - listing, adding and removing - is a stopgap before we get KOKO-1952 implemented?

Reply:
Good call - added a tracking issue for that as well: https://github.com/Kong/gateway-operator/issues/634